packages/react-ui/src/widgets/TimeSeriesWidgetUI/TimeSeriesWidgetUI.js

```diff
@@ -328,13 +336,15 @@ function TimeSeriesWidgetUIContent({
           onClick={handleClear}
           underline='hover'
         >
-          Clear
+          {intlConfig.formatMessage({ id: 'c4r.widgets.timeSeries.clear' })}
         </Link>
       )}
     </>
   );
 
-  const controls = showControls && <TimeSeriesControls data={data} stepSize={stepSize} />;
+  const controls = showControls && (
+    <TimeSeriesControls data={data} stepSize={stepSize} intl={intl} />
```

Comment:
maybe we could have `context` have access to `intl` where is needed, instead of applying props drilling?

Reply:
yes, we could, sounds good 👍

Reply:
Already applied to PR, replacing HOC with useIntl hook